lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1134,14 +1134,24 @@ def is_service_active(self, service) -> bool:
         Returns:
             service active status
         """
+        service_name = self.get_service_name_from_init_system(service)
```

Comment:
Just a heads up: For systemd services this check can be problematic, because services caught in a re-start loop can report 'active' by chance. Does this also holds for the docker case? Do we know that calling places are aware of this?

Reply:
We currently do not check the systemd services for is-active here, but rather wait the empirically obtained sleep time as we always did and do not call this function during restarts of systemd services. It is in fact currently only used to enable/disable the health service at the beginning of a test.
In the docker case we explicitly check for the health status to be healthy. Container in a restart loop do report restarting and the check fails / runs into the maximum waiting time.